operator/operator_test.go

```diff
@@ -112,7 +112,7 @@ type environment struct {
 	enclaveParameters tee.Parameters
 }
 
-const blockTime = 1 // block mining interval in seconds
+const blockTime = 2 // block mining interval in seconds
```

Comment:
Can we change the test so that we subscribe to new heads to synchronize with the ganache's blocks? This way we can reuse the same test for testnets.

And I think that it makes sense to have the ganache use a block time of 1 sec so that tests run faster.

Reply:
Yea we have a `WaitForBlock` function in the ethereum client which should work.

Reply:
Changed it back to 1 sec.